### PR TITLE
Added new rule: jsx-no-multiline-js

### LINKED ISSUE #4: New Rule: jsx-no-multiline-js
Ban multiline JSX expressions in JSX.

### PR DESCRIPTION
**Pull request by davidlipo**
Fixes #4

### CODE REVIEW

**Comment by adidahiya** [comment]
See here for the proper syntax for closing issues via PR descriptions: https://github.com/blog/1506-closing-issues-via-pull-requests

File: src/rules/jsxNoMultilineJsRule.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as ts from "typescript";
+import * as Lint from "tslint/lib/lint";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static FAILURE_STRING = "JSX expressions should not be multiline";
```
Comment (adidahiya) [inline src/rules/jsxNoMultilineJsRule.ts]: A more consistent error message would be "Multiline JS expressions inside JSX are forbidden"

**Comment by adidahiya** [comment]
Needs more thorough testing, like multiline expressions inside JSX attributes.

File: src/rules/jsxNoMultilineJsRule.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
+        const walker = new JsxNoMultilineJsWalker(sourceFile, this.getOptions());
+        return this.applyWithWalker(walker);
+    }
+}
+
+class JsxNoMultilineJsWalker extends Lint.RuleWalker {
+    protected visitJsxExpression(node: ts.JsxExpression) {
+        if (node.getText().indexOf("\n") > -1) {
```
Comment (JKillian) [inline src/rules/jsxNoMultilineJsRule.ts]: This is definitely simpler than how I did `isMultiline`, I kinda like it.

**Comment by JKillian** [comment]
Looks good to me!

**Comment by adidahiya** [comment]
@davidlipo please submit another PR to add this rule to the README

**Pull request merged**